src/euphorie/client/browser/webhelpers.py

```diff
@@ -731,6 +740,66 @@ def show_logo(self):
         """
         return True
 
+    @memoize_contextless
+    def date_picker_i18n_json(self):
+        """ Taken from:
+        https://github.com/ploneintranet/ploneintranet/blob/master/src/ploneintranet/layout/browser/date_picker.py  # noqa: E501
+
+        Use this like:
+        <input class="pat-date-picker"
+               ...
+               data-pat-date-picker="...; i18n: ${portal_url}/@@date-picker-i18n.json; ..."
+               />
+        """
+        json = dumps(
+            {
+                "previousMonth": self.translate(pae_message("prev_month_link")),
+                "nextMonth": self.translate(pae_message("next_month_link")),
+                "months": [
+                    self.translate(pl_message(month))
+                    for month in [
+                        "month_jan",
+                        "month_feb",
+                        "month_mar",
+                        "month_apr",
+                        "month_may",
+                        "month_jun",
+                        "month_jul",
+                        "month_aug",
+                        "month_sep",
+                        "month_oct",
+                        "month_nov",
+                        "month_dec",
+                    ]
+                ],
+                "weekdays": [
+                    self.translate(pl_message(weekday))
+                    for weekday in [
+                        "weekday_sun",
+                        "weekday_mon",
+                        "weekday_tue",
+                        "weekday_wed",
+                        "weekday_thu",
+                        "weekday_fri",
+                        "weekday_sat",
+                    ]
+                ],
+                "weekdaysShort": [
+                    self.translate(pl_message(weekday_abbr))
+                    for weekday_abbr in [
+                        "weekday_sun_abbr",
+                        "weekday_mon_abbr",
+                        "weekday_tue_abbr",
+                        "weekday_wed_abbr",
+                        "weekday_thu_abbr",
+                        "weekday_fri_abbr",
+                        "weekday_sat_abbr",
+                    ]
+                ],
+            }
+        )
+        return json
+
```

Comment:
This seems to be very similar to the `date-picker-i18n.json` grok view that we already have:
https://github.com/euphorie/Euphorie/blob/master/src/euphorie/client/utils.py#L193

I am in favour of retiring grok :-) But could you please enhance this PR by removing the old grok view and adjusting all places where is `date-picker-i18n.json` is used?

Reply:
I completely missed it